### PR TITLE
chore: bump kommander/opsportal to image 1.162.0

### PR DESCRIPTION
**Pull request by natmegs**

### CODE REVIEW

File: stable/kommander/values.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  tag: 1.153.1
+  tag: 1.162.0
   pullPolicy: IfNotPresent
 replicas: 1
 logoutRedirectPath: /ops/landing
 clusterPollingInterval: 3000
 hostPollingInterval: 3000
 clusterPollingThrottleFactor: 1
-kubernetesVersionsSelection: '[{"kubernetesVersion":"1.15.5","konvoyVersion":"v1.3.0-beta6","kubeaddonsVersion":"kommander-beta6"}]'
+kubernetesVersionsSelection: '["1.15.5", "1.15.4"]'
```
Comment (natmegs) [inline stable/kommander/values.yaml]: @DanielMSchmidt Can we confirm that this is the expected format and these are the correct values?

Comment (DanielMSchmidt) [reply stable/kommander/values.yaml]: Yes, that is the new format 👍

Comment (natmegs) [reply stable/kommander/values.yaml]: @jimmidyson can you confirm that these are the supported versions in KCL? They are an assumption based on https://github.com/mesosphere/kubeaddons-configs/blob/master/templates/kommander.yaml#L49

Comment (jimmidyson) [reply stable/kommander/values.yaml]: For now, yeah, although we do also support 1.16.1, we need a konvoy release to support that properly.

Comment (natmegs) [reply stable/kommander/values.yaml]: @jimmidyson will that support be available in the beta7 konvoy release?

**Comment by jimmidyson** [comment]
Yes it should be

File: stable/kommander/values.yaml (hunk 1 above)
Comment (jimmidyson) [reply stable/kommander/values.yaml]: Yes

**Pull request merged**